Bugzilla/Product.pm

```diff
@@ -47,6 +47,7 @@ use constant DB_COLUMNS => qw(
    isactive
    defaultmilestone
    allows_unconfirmed
+   bug_comment_template
```

Comment:
I'm concerned this will increase memory usage.
I think we can lazy-load this like the securemail extension with the public_key
field.

First remove it from DB_COLUMNS and then:

```perl
# We want to lazy-load the public_key.
sub bug_comment_template {
    my $self = shift;
    if (!exists $self->{public_key}) {
        ($self->{bug_comment_template}) = Bugzilla->dbh->selectrow_array(
            "SELECT bug_comment_template FROM products WHERE id = ?",
            undef,
            $self->id
        );
    }
    return $self->{bug_comment_template};
}
```